logstash-core/locales/en.yml

```diff
@@ -82,9 +82,9 @@ en:
     runner:
       short-help: |-
         usage:
-          bin/logstash -f CONFIG_PATH [-t] [-r] [--quiet|verbose|debug] [-w COUNT] [-l LOG]
-          bin/logstash -e CONFIG_STR [-t] [--quiet|verbose|debug] [-w COUNT] [-l LOG]
-          bin/logstash -i SHELL [--quiet|verbose|debug]
+          bin/logstash -f CONFIG_PATH [-t] [-r] [--error|info|debug|warn] [-w COUNT] [-l LOG]
+          bin/logstash -e CONFIG_STR [-t] [--error|info|debug|warn] [-w COUNT] [-l LOG]
+          bin/logstash -i SHELL [--error|info|debug|warn]
```

Comment:
can you add `fatal` and `trace` levels ?
can you put `warn` level before `info` level ?